tests/utils_tests/test_deprecation.py

```diff
@@ -14,3 +14,14 @@ def test_false(self):
         self.assertEqual(CallableFalse, False)
         self.assertFalse(CallableFalse != False)  # noqa: E712
         self.assertNotEqual(CallableFalse, True)
+
+    def test_or(self):
+        self.assertTrue(CallableTrue | CallableTrue)
```

Comment:
I think we should use assertions like `self.assertIs(CallableTrue | CallableTrue, True)`. That'll reveal that `__or__` should return `bool` rather than `CallableBool`.